backend/Application/Api/GraphQL/Import/BakerImportHandler.cs

```diff
@@ -351,13 +359,19 @@ private async Task WorkAroundConcordiumNodeBug225(BlockInfo blockInfo, ImportSta
             if (txEvent is ConcordiumSdk.NodeApi.Types.BakerRemoved bakerRemoved)
             {
                 var pendingChange = await pendingChangeStrategy.SetPendingChangeOnBaker(bakerRemoved);
-                UpdateNextPendingBakerChangeTimeIfLower(pendingChange.EffectiveTime, importState);
+                if (pendingChange is not null)
+                {
+                    UpdateNextPendingBakerChangeTimeIfLower(pendingChange.EffectiveTime, importState);
+                }
```

Comment:
Do we not always expect the pending change to not be null?

Would it make sense to log the error then?

Reply:
done